sympy/core/containers.py

```diff
@@ -50,6 +50,12 @@ class Tuple(Basic):
     def __new__(cls, *args, **kwargs):
         if kwargs.get('sympify', True):
             args = (sympify(arg) for arg in args)
+        #if kwargs.get('denest_lists', False):
```

Comment:
This is to be removed before merging, but shows that currently there are no longer any `Tuple` with `list` args being created during the testing.